deploifai/dataset/pull.py

```diff
@@ -0,0 +1,42 @@
+from pathlib import Path
+
+import click
+
+from deploifai.clouds.utilities.data_storage.handler import DataStorageHandlerEmptyFilesException, \
+    DataStorageHandlerTargetNotFoundException
+from deploifai.context import DeploifaiContextObj, pass_deploifai_context_obj, is_authenticated, dataset_found
+from deploifai.core.data_storage import DataStorage
+
+
+@click.command()
+@click.argument("target", required=False)
+@pass_deploifai_context_obj
+@is_authenticated
+@dataset_found
+def pull(context: DeploifaiContextObj, target: str = None):
+    """
+    Download files from the cloud to local.
+    """
+
+    api = context.api
+    dataset_id = context.dataset_config["DATASET"]["id"]
+
+    data = context.api.get_data_storage_info(dataset_id)
+    if len(data) == 0:
+        click.secho("No dataset information extracted", fg="yellow")
+        click.secho("Please run deploifai dataset init again")
```

Comment:
Probably should just be `Dataset has not been initialized, please run deploifai dataset init`.

Reply:
This only happens if they have tampered with the dataset info or no longer have access to that dataset
So unsure how to handle this